packages/shorebird_cli/lib/src/auth/auth.dart

```diff
@@ -45,18 +46,34 @@ typedef OnRefreshCredentials = void Function(
   oauth2.AccessCredentials credentials,
 );
 
-class AuthenticatedClient extends http.BaseClient {
+class LoggingClient extends http.BaseClient {
+  LoggingClient({
+    required http.Client httpClient,
+    required Logger logger,
+  })  : _baseClient = httpClient,
+        _logger = logger;
+
+  final http.Client _baseClient;
+  final Logger _logger;
+
+  @override
+  Future<http.StreamedResponse> send(http.BaseRequest request) async {
+    _logger.detail('$request');
```

Comment:
```suggestion
    _logger.detail('[HTTP] $request');
```

nit but thoughts on tagging logs so that it's easier to filter by specific types of logs?

Reply:
sgtm, done